backend/model/Hugs.js

```diff
@@ -261,6 +262,7 @@ const UpdateHugAPI = {
       .collection("notifications");
     requestIdRef = currUserNotifRef.where("hug_id", "==", hugId);
     Notifications.NotificationsAPI.deleteNotification(requestIdRef);
+     */
   },
 
   updateUserHugCount: function (hugId) {
```

Comment:
Should this be async?

Reply:
this is fine even though it deviates from the other functions bc there is no await statement, instead this is handling async logic with .then()